notebooks/concurrent_tasks.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>I would change</p><p> </p><blockquote>In this workbook, we show how you can perform tasks concurrently by wrapping multiple components into one and using asyncio.</blockquote><p> </p><p>into</p><p> </p><blockquote>In this cookbook, we show how you can wrap multiple components into one that will run them concurrently in different threads.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>done.</p>